src/components/AddContactPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ContactForm from './ContactForm';
+
+const propTypes = {
+  state: PropTypes.objectOf(PropTypes.shape({
```

Comment:
Very wrong approach. Never combine different props to generic object, should be two separate props

Reply:
divided to separate props (searchValue and contactList)